app/(gcforms)/[locale]/(form administration)/form-builder/[id]/settings/api-integration/page.tsx

```diff
@@ -0,0 +1,137 @@
+import { redirect } from "next/navigation";
+import { Metadata } from "next";
+import Markdown from "markdown-to-jsx";
+
+import { serverTranslation } from "@i18n";
+import { authCheckAndThrow } from "@lib/actions";
+import { ApiKeyButton } from "./components/ApiKeyButton";
+import { ApiKeyDialog } from "../../components/dialogs/ApiKeyDialog/ApiKeyDialog";
+import { DeleteApiKeyDialog } from "../../components/dialogs/DeleteApiKeyDialog/DeleteApiKeyDialog";
+import { AuthenticatedPage } from "@lib/pages/auth";
+
+import { getCurrentThrottlingRate } from "./components/throttlingRate/actions";
+import { LinkButton } from "@serverComponents/globals/Buttons/LinkButton";
+import { getFullTemplateByID } from "@lib/templates";
+import { unConfirmedResponsesExist } from "../actions";
+import { UnconfirmedApiKeyDialog } from "../../components/dialogs/ApiKeyDialog/hasUnconfirmedApiKeyDialog";
+
+import { ManageApiThrottle } from "./components/throttlingRate/ManageApiThrottle";
+
+export async function generateMetadata(props: {
+  params: Promise<{ locale: string }>;
+}): Promise<Metadata> {
+  const params = await props.params;
+
+  const { locale } = params;
+
+  const { t } = await serverTranslation("form-builder", { lang: locale });
+  return {
+    title: `${t("settings.apiIntegration.navigation.title")} — ${t("gcForms")}`,
+  };
+}
+
+export default AuthenticatedPage(
+  async (props: { params: Promise<{ id: string; locale: string }> }) => {
+    const params = await props.params;
+
+    const { locale } = params;
+
+    const { id } = params;
+
+    const limit = await getCurrentThrottlingRate(id);
+    let rate = null;
+
+    if ("error" in limit) {
+      // no-op no need to handle as this is just to show or hide the suoport text
```

Comment:
Note: if we error here --- we're safe to continue as this just alters if the contact support to up your limit content shows  ... in this case it will show.